terraform/modules/prod/variables.tf

```diff
@@ -0,0 +1,33 @@
+variable "prefix" {
```

Comment:
I do like this structure, it's annoying that you need to duplicate most of the environment variables, but on the other hand it's no terrible thing that you need to think actively about each value in each environment